openfecwebapp/api_caller.py

```diff
@@ -53,12 +53,14 @@ def load_legal_search_results(query, query_type='all', ao_no=None, ao_name=None,
                                 ao_min_date=None, ao_max_date=None,
                                 offset=0, limit=20):
     filters = {}
-    if query:
-        filters['q'] = query
+    if query or query_type == 'advisory_opinions':
```

Comment:
So, it's the behavior I believe we want in _this_ scenario, but I'll defer to @anthonygarvan or @vrajmohan as to if this is wise on the whole.